### ISSUE TITLE
response 403

### ISSUE THREAD

**Issue by BWZhang-NLP** [cooolr/chatgpt_plus_python_api#1]
get_authorization error:  requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

'requests.get(url, headers=headers)' ->  response 403

**Comment by cooolr** [comment]
`puid` `cf_clearance` `session_token` Have you filled in these three parameters?

**Comment by BWZhang-NLP** [comment]
Yes,I have filled the three parameters. It suddenly reported an error yesterday afternoon. So I print the obtained authentication result,the response status is  403.

**Comment by cooolr** [comment]
you can access https://chat.openai.com/api/auth/session to get access_token, i check it and fix later.

**Comment by BWZhang-NLP** [comment]
Ok,thank you.

**Comment by GreyDGL** [comment]
Same problem here. Seems like some other cookies are required to access the API now.
